docs/.vale.ini

```diff
@@ -0,0 +1,24 @@
+StylesPath = .vale/writing-styles
+
+MinAlertLevel = warning
+
+[formats]
+mdx = md
+
+SkippedScopes = script, style, pre, figure, code, tt, tr, td,
+
+[*.md]
+BasedOnStyles = expo-docs
+
+# Ignore code surrounded by backticks or plus sign, parameters defaults, URLs.
+TokenIgnores = (\x60[^\n\x60]+\x60), ([^\n]+=[^\n]*), (\+[^\n]+\+), (http[^\n]+\[)
+
+# Temporarily disable consistency checks when testing
+# expo-doc.FileName = NO
+
+# TODO: (@amandeepmittal) to remove these after testing
+# expo-docs.FirstPerson=NO
+# expo-docs.Ellipses=NO
+# expo-docs.AppStores=NO
+# expo-docs.Latin=NO
+# expo-docs.PlatformsOrder=NO
```

Comment:
```suggestion
# Temporarily disable consistency checks when testing
# expo-doc.FileName = NO

# TODO: (@amandeepmittal) to remove these after testing
# expo-docs.FirstPerson=NO
# expo-docs.Ellipses=NO
# expo-docs.AppStores=NO
# expo-docs.Latin=NO
# expo-docs.PlatformsOrder=NO
```
Can we remove the commented out lines, or we plan to re-enable them soon?

Reply:
I was planning to use them to fix these warnings but I think right now, we can remove this. Thanks for catching this!